tests/pageObjects/infinite-scroller.ts

```diff
@@ -183,6 +183,10 @@ export class InfiniteScroller {
     }
   }
 
+  async checkFirstItemResult() {
```

Comment:
suggestion: let's describe the goal along w/ responsibility
```suggestion
  async validateFirstResult() {
```